Template/Tag/CustomHtmlTag.web.js

```diff
@@ -125,6 +125,13 @@
                 newScript.setAttribute('type', element.getAttribute('type'));
             }
 
+            // If this script is executing using a nonce, set the nonce on its children too.
+            var scriptWithNonce = TagManager.dom.bySelector('script[nonce]');
+            if (Array.isArray(scriptWithNonce) && scriptWithNonce.length) {
```

Comment:
@snake14 This check is failing for me since the type is `object` for me
I removed the `Array.isArray(scriptWithNonce)` check to make the if loop work

Reply:
That's strange. It uses querySelectorAll which is supposed to return an instance of NodeList, which implements Array. I checked it in both Brave (Chromium) and Firefox and both consoles show as type object, but that it implements the Array prototype. Did you have the nonce attribute set in at least one of your script elements on your test page?

Reply:
Yes I had `nonce` attribute setup on the script element